utilities.py

```diff
@@ -0,0 +1,11 @@
+def flat(data: dict, subtest: str):
```

Comment:
Add a PEP8-styled block comment that describes what this function does as well as the type of the data that is expected. (You can also remove the dict and str specifications from the arguments once you've added the comment).